nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/config/NepomukExceptionConfig.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.diging.gilesecosystem.nepomuk.core.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import edu.asu.diging.gilesecosystem.nepomuk.core.service.properties.Properties;
+import edu.asu.diging.gilesecosystem.septemberutil.service.impl.SystemMessageHandler;
+import edu.asu.diging.gilesecosystem.util.properties.IPropertiesManager;
+
+/**
+ * Class to initialize message handler class to send exception as kafka topic.
+ * 
+ * @author snilapwa
+ *
+ */
+@Configuration
+public class NepomukExceptionConfig {
```

Comment:
lets just call that NepomukConfig since we'll probably add other beans here as well in the future that have nothing to do with exceptions.